Neos.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/FileUpload.js

```diff
@@ -96,9 +97,11 @@ function(Ember, $, template, plupload, Notification, Configuration, I18n) {
 				uploader.splice();
 			});
 
-			this._uploader.bind('BeforeUpload', function(uploader, file) {
+			this._uploader.bind('BeforeUpload', function(uploader) {
 				uploader.settings.multipart_params['__csrfToken'] = Configuration.get('CsrfToken');
-				uploader.settings.multipart_params['__siteNodeName'] = $('link[rel="neos-site"]').data('node-name');
+				uploader.settings.multipart_params['node'] = InspectorController.nodeSelection.get('selectedNode.nodePath');
+				uploader.settings.multipart_params['propertyName'] = that.get('property');
```

Comment:
Is it OK to retrieve the current property name like this?  It seems to work at least ;)

Reply:
yup that's the right way to do it! see https://github.com/neos/neos-development-collection/blob/master/Neos.Neos/Resources/Public/JavaScript/Content/Inspector/PropertyEditor.js#L72